backend/src/dependencies/install_required_deps.py

```diff
@@ -0,0 +1,46 @@
+import subprocess
+from json import loads as json_parse
+
+from .store import installed_packages, python_path
+from .versioned_dependency_helpers import install_version_checked_dependency
+
+# Get the list of installed packages
+# We can't rely on using the package's __version__ attribute because not all packages actually have it
+try:
+    pip_list = subprocess.check_output(
+        [python_path, "-m", "pip", "list", "--format=json"]
+    )
+    # set_installed_packages({p["name"]: p["version"] for p in json_parse(pip_list)})
+    for p in json_parse(pip_list):
+        installed_packages[p["name"]] = p["version"]
+except Exception as e:
+    installed_packages = {}
+
+
+deps = [
+    {
+        "package_name": "sanic",
+        "version": "23.3.0",
+    },
+    {
+        "package_name": "Sanic-Cors",
+        "version": "2.2.0",
+    },
+    {
+        "package_name": "numpy",
+        "version": "1.23.2",
+    },
+    {
+        "package_name": "opencv-python",
+        "version": "4.7.0.68",
+    },
+    {
+        "package_name": "Pillow",
+        "version": "9.2.0",
+    },
```

Comment:
I think we should move those 2 into `chainner_standard` or other deps.

Reply:
Can't because the API needs them. I explained this in discord. To make that not happen I'd need to move around a ton of files and i don't want to do that in this PR